f2/src/forms/ContactInfoForm.js

```diff
@@ -55,17 +61,19 @@ export const ContactInfoForm = (props) => {
                     title={<Trans id="contactinfoPage.hasValidationErrors" />}
                   />
                 )}
-                {fyiForm ? (
+                <br />
```

Comment:
We shouldn't use `<br/>` tag unless absolutely necessary. Instead a styled row should be used. See `<FormRow>` in f2\src\components\formik\row\index.js.

Reply:
Resolved